gbpservice/nfp/service_vendor_agents/vyos/agent/src/config_server/server.py

```diff
@@ -1,32 +1,31 @@
-# One Convergence, Inc. CONFIDENTIAL
-# Copyright (c) 2012-2015, One Convergence, Inc., USA
-# All Rights Reserved.
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
```

Comment:
Check if this file is used. We need to check if there are unused files and they should not be submitted.